app/controllers/pages_controller.rb

```diff
@@ -9,8 +9,10 @@ def profile
     # @reviews = current_user.reviews #non piace a giulia
     @lessons = current_user.lessons # lessons booked as a user
     if current_user.is_mentor
-      @mentor_lesson = current_user.lesson # your lesson as a mentor
-      @mentor_bookings = @mentor_lesson.bookings # mentor bookings of your lesson
+        @mentor_lesson = current_user.lesson # your lesson as a mentor
+        if @mentor_bookings != nil
+        @mentor_bookings = current_user.lesson.bookings # mentor bookings of your lesson
```

Comment:
can be refactored with ternary opreration. 
example 
@lesson = current_user.lesson
@bookings = @lesson ? @lesson.bookings : current_user.bookings

Reply:
(disregard the caps)

Reply:
REFACTORED:
if current_user.is_mentor
      @mentor_lesson = current_user.lesson
      @mentor_bookings = @mentor_lesson ? @mentor_lesson.bookings : current_user.bookings
    end

Reply:
Tested it on a mentor profile with no bookings and no lessons, it works.